packages/x-charts/src/Toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+
+const ToolbarRoot = styled('div', {
+  name: 'MuiChartsToolbar',
+  slot: 'Root',
+})(({ theme }) => ({
+  flex: 0,
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'end',
+  gap: theme.spacing(0.25),
+  padding: theme.spacing(0.5),
+  minHeight: 44,
+  boxSizing: 'border-box',
+  border: `1px solid ${(theme.vars || theme).palette.divider}`,
+  borderRadius: 4,
+}));
+
+export interface ToolbarProps extends React.PropsWithChildren {}
+
+export function Toolbar({ children }: ToolbarProps) {
```

Comment:
Should the toolbar be a pro offering?

Reply:
I don't think so. Some buttons inside the toolbar might only be enabled in case some pro features are enabled, but I think the toolbar itself should be available in the community package. We might have other interactions that require the toolbar but aren't pro/premium.

Nevertheless, @alexfauquette what do you think?

Reply:
I'm fine with keeping the toolbar in the MIT package. But I don't see any button to add that are not pro features. I can think about export, zoom manipulation or annotation

Reply:
Yeah, I added an empty toolbar in the community package so users can add their own buttons.